sphinx/util/typing.py

```diff
@@ -178,14 +178,26 @@ def _is_annotated_form(obj: Any) -> TypeGuard[Annotated[Any, ...]]:
     return typing.get_origin(obj) is Annotated or str(obj).startswith('typing.Annotated')
 
 
+def _is_unpack_form(obj: Any) -> bool:
+    """Check if the object is :class:`typing.Unpack` or equivalent."""
+    if sys.version_info >= (3, 11):
+        from typing import Unpack
+
+        return typing.get_origin(obj) is Unpack
```

Comment:
Ok, this one might be indeeed easier. Should we use a typeguard for this one as well even though it might be fooled? (I've done it for the Annotated type but I'm not sure how well mypy will be happy if fool it).

Reply:
I tried with a typeguard, but mypy kept complaining -- I am waiting for mypy 1.10, which will support ``TypeIs``.

A